string-manipulation-2/src/first-chars.js

```diff
@@ -1,14 +1,14 @@
 /* exported firstChars */
 function firstChars(length, string) {
   var output = '';
+  if (string === '') {
+    output += '';
```

Comment:
This block shouldn't be necessary. If the string is empty, its length will be `0` which means that the loop below wouldn't start, leaving you with the same result of returning `output` which will contain an empty string.